eq_cir_management_ui/errors/routes.py

```diff
@@ -0,0 +1,63 @@
+"""Errors routes."""
+
+from flask import Blueprint, render_template, request
+from structlog import get_logger
+from werkzeug.exceptions import (
+    Forbidden,
+    InternalServerError,
+    NotFound,
+    Unauthorized,
+)
+
+logger = get_logger()
+
+errors_blueprint = Blueprint("errors", __name__)
+
+
+def log_exception(exception: Exception, status_code: int) -> None:
+    """Log the exception with the appropriate log level based on the status code."""
+    log = logger.warning if status_code < 500 else logger.error
+
+    log(
+        "an error has occurred",
+        exc_info=exception,
+        url=request.url,
+        status_code=status_code,
+    )
+
+
+@errors_blueprint.app_errorhandler(401)
```

Comment:
Do we want to handle other errors (400 "Bad Request", 405 "Method not allowed") and use existing templates for those like we do in Runner?